buildSrc/build.gradle

```diff
@@ -3,36 +3,35 @@ plugins {
     id "groovy-gradle-plugin"
 }
 
+Properties properties = new Properties()
+FileInputStream input = new FileInputStream(file('../gradle.properties'))
+properties.load(input)
+input.close()
```

Comment:
I think this could be simplified to:
```groovy
Properties properties = new Properties()
file('../gradle.properties').withInputStream { input ->
    properties.load(input)
}
```

Reply:
Updated.